demos/bunny/wasmmark/build.zig

```diff
@@ -1,19 +1,27 @@
 const std = @import("std");
 
-pub fn build(b: *std.build.Builder) void {
-    const mode = b.standardReleaseOptions();
+pub fn build(b: *std.Build) !void {
+    const optimize = b.standardOptimizeOption(.{});
+
+    const lib = b.addSharedLibrary(.{
+        .name = "cart",
+        .root_source_file = .{ .path = "src/main.zig" },
+        .target = .{ .cpu_arch = .wasm32, .os_tag = .wasi },
+        .optimize = optimize,
+    });
 
-    const lib = b.addSharedLibrary("cart", "src/main.zig", .unversioned);
-    lib.setBuildMode(mode);
-    lib.setTarget(.{ .cpu_arch = .wasm32, .os_tag = .freestanding });
     lib.import_memory = true;
+    lib.stack_size = 8192;
     lib.initial_memory = 65536 * 4;
     lib.max_memory = 65536 * 4;
+
     lib.export_table = true;
+
     // all the memory below 96kb is reserved for TIC and memory mapped I/O
     // so our own usage must start above the 96kb mark
     lib.global_base = 96 * 1024;
-    lib.stack_size = 8192;
-    lib.export_symbol_names = &[_][]const u8{ "TIC", "OVR", "BDR", "SCR", "BOOT" };
-    lib.install();
+
+    lib.export_symbol_names = &[_][]const u8{ "TIC", "OVR", "BDR", "BOOT" };
```

Comment:
Has SCR been entirely removed now or just deprecated?  If the latter we many want to leave it here for backwards compatibility...

Reply:
I don't think someone used it, first of all, it was initially called SCN, and second, most langs don't support SCN (only Lua and JS). We can remove it without any doubts.